### ISSUE TITLE
EditPlaytime is not working properly

### ISSUE THREAD

**Issue by naikyys** [Zrips/CMI#7652]
### Description of Issue

Hi, I'm struggling with a problem regarding changing the playtime. For some people it doesn't cause a problem, but for others it does. I don't know what it is, it's just that when I try to add time by 5000 for example, it goes back to the old value and the playtime doesn't move.

Objection: Playtime is not taken from minecraft statistics, its set to false in CMI config

It looks like this:
![image (1)](https://user-images.githubusercontent.com/67881641/222941426-f488b362-0dc7-4abf-ac93-a945ed8be93d.png)

### Version Information

```shell
CMI: 9.3.1.6
CMILib: 1.2.4.5
Server: Purpur 1.18.2-R0.1-SNAPSHOT
```

### Agreements

- [x] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by Zrips** [comment]
Is this issue happen for online and offline players or is this only limited to one of those states of the player?

**Comment by naikyys** [comment]
> Is this issue happen for online and offline players or is this only limited to one of those states of the player?

Okay. I found out even more weird problem. Maybe this is the reason why it's working. Like I tried in console on myself ``/editplaytime naikk_ add 20000``** but I had to be OFFLINE, if I was online it WAS NOT WORKING**.

So the problem is actually that its not working if the player is online and also this problem, the player does not exist? What the hell?

![r35EEm4](https://user-images.githubusercontent.com/67881641/223425680-cea58d39-d4b9-42e4-a0f8-3b42d002326f.png)

**Comment by Zrips** [comment]
Issue with missing player will be fixed, good catch on this one.
For player playtime edit, by any chance you are in vanish mode and have option enabled which stops your playtime?

**Comment by naikyys** [comment]
> Issue with missing player will be fixed, good catch on this one. For player playtime edit, by any chance you are in vanish mode and have option enabled which stops your playtime?

Yes. The setting that stops my playtime is set to true in ``/vanishedit``.  But even if it's set to false, this issue is still happening.
![yu4sGXZ](https://user-images.githubusercontent.com/67881641/223454293-b1d19f7a-7ac4-449a-8cb6-a34471f20dc6.png)

But if I turn off vanish completely - I became visible, its working fine. So it looks like the problem is just in vanish mode which is problem but not really 😄
![ecbOK91](https://user-images.githubusercontent.com/67881641/223455130-fb8cc81e-39b1-4a1e-bf7b-01d4b2704ad2.png)

**Comment by TomLewis** [comment]
Im also having issues, where im updating a playtime for an offline player. Check it in game with /playtime <username> and also /stats and then when they come online, it reverts itself! Ive even reloaded the plugin to make sure its still there.. no idea, I just cant get the change to set/save.

**Comment by TomLewis** [comment]
Update!

It works if the player is online. 
It looks like it works when they are offline, until they come on, then it reverts!

**Comment by TomLewis** [comment]
this was never fixed
